sdk/iOS/src/MSQueryResult.h

```diff
@@ -10,7 +10,7 @@
 ///@{
 
 /// The result from a successful read
-@property (nonatomic, strong, readonly)           NSArray *items;
+@property (nonatomic, strong, readonly, nonnull)           NSArray<NSDictionary *> *items;
```

Comment:
granted perhaps we should add some errors on table reads who don't return arrays that we can parse, as I think non arrays are what result in nil...